cubic-server/Dimension.cpp

```diff
@@ -159,7 +159,7 @@ void Dimension::loadOrGenerateChunk(int x, int z, std::shared_ptr<Player> player
                 const Vector3<double> chunkPos = {(double) x * 16, pos.y, (double) z * 16};
                 current_min = std::min(current_min, pos.distance(chunkPos));
             }
-            return static_cast<size_t>(std::ceil(current_min));
+            return static_cast<int>(std::ceil(current_min));
```

Comment:
Why change it to an int? Is there a reason? Because `current_min` cannot be negative in the first place.

Reply:
It's not a problem with current_min, but the threadpool job priority function which take an int, and it may throw a warning because of the narrowing.